dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -102,7 +102,7 @@ internal class RumSessionScope(
             )
             onViewDisplayed(event, viewScope, actualWriter)
             activeChildrenScopes.add(viewScope)
-        } else if (activeChildrenScopes.size == 0) {
+        } else if (activeChildrenScopes.count { it.isActive() } == 0) {
```

Comment:
Should we rename `activeChildrenScopes` field somehow? Or use other naming for `isActive`? Because with that change it introduces some ambiguity: basically this line says that `activeChildrenScopes` can contain some scope which is not actually `active` (and `activeChildrenScopes` was filtered just few lines above).

Or we can introduce set of states for `RumScope`: `active`, `stopping`, `stopped` or something like this. WDYT?

Reply:
Indeed I'll rename `activeChildrenScopes` -> `childrenScopes`